crates/sui-protocol-config/src/lib.rs

```diff
@@ -1260,6 +1261,11 @@ impl ProtocolConfig {
                 cfg.feature_flags.simplified_unwrap_then_delete = true;
                 cfg
             }
+            17 => {
+                let mut cfg = Self::get_for_version_impl(version - 1, chain);
+                cfg.execution_version = Some(1);
```

Comment:
so we run latest instead of v0? a bit confused here....

Reply:
Yes, we run `latest` -- this is so that we don't disrupt other folks who are working on things that need to go out, that were working off of `latest` before I cut this release.  I.e. all we're doing in this PR is building in the mechanism for versioning, but we are not using it to change how we release to production, only how we preserve existing behaviour.